packages/wallet/api/test/test-lib-wallet.js

```diff
@@ -914,7 +914,7 @@ test('lib-wallet offer methods', async t => {
         },
         inviteHandleBoardId: 'board0257',
         meta: {
-          id: 9,
+          id: 8,
```

Comment:
This change made blindly. Reviewers, how should I figure out whether I should worry about this change?